heudiconv/info.py

```diff
@@ -43,7 +43,10 @@
     'extras': [
         'duecredit',  # optional dependency
     ],  # Requires patched version ATM ['dcmstack'],
-    'datalad': ['datalad >=%s' % MIN_DATALAD_VERSION]
+    'datalad': ['datalad >=%s' % MIN_DATALAD_VERSION],
+    'dcm2bids': [
```

Comment:
What about `physio` for this dependency group?

Reply:
Good idea!